phys2bids/phys2bids.py

```diff
@@ -107,7 +88,10 @@ def use_heuristic(heur_file, sub, ses, filename, outdir):
     heur = utils.load_heuristic(heur_file)
     name = heur.heur(filename[:-4], name)
 
-    heurpath = fldr + '/' + name + '_physio'
+    if record_label:
+        recording = f'_recording-{record_label}'
+
+    heurpath = fldr + '/' + name + recording + '_physio'
```

Comment:
Right, that's "Good First Issue" #40 . @eurunuela had similar concerns.
Can we keep it in that issue and go on with this PR separately? Otherwise I'll do all of that one.

Reply:
Sorry, wasn't referencing the issues when looking at this. That's fine!